apps/docs/src/guide/wallets/wallet-transferring.md

```diff
@@ -1,14 +1,59 @@
 # Wallet Transferring
 
-This guide demonstrates how to transfer assets between accounts and contracts, and how to validate your balance prior to a transfer.
+This guide provides instructions for transferring assets between wallets and contracts using the SDK. It includes methods for initiating transfers, handling transaction requests, and ensuring sufficient balance before transfers.
 
-## Transferring Between Wallets
+## Transferring Assets Between Accounts
 
-Transferring assets between wallets is really simple within the SDK.
+The `account.transfer` method initiates a transaction request that transfers an asset from one wallet to another. This method requires three parameters:
+
+1. The receiver's wallet address.
+2. The amount of the asset to be transferred.
+3. The ID of the asset to be transferred (Optional - Defaults to the base asset id).
+
+Upon execution, this function returns a promise that resolves to a transaction response. To wait for the transaction to be processed, call `response.wait()`.
```

Comment:
Will we stay with `.wait()` or `.waitForResult()`? #1146

Reply:
@petertonysmith94 I reckon we should use `waitForResult` as it is the most used within our code examples.